### PR TITLE
feat: updates for v2 of tableland sdk

### PR DESCRIPTION
**Pull request by carsonfarmer**
After we merge @tableland/sdk and bump its version, we'll want to update it here, and then bump this version as well.

### CODE REVIEW

File: src/commands/read.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      description: "Output format. One of 'raw', 'tabular', or 'objects'.",
+      default: "raw",
+    }) as yargs.Argv<Options>;
+
+export const handler = async (argv: Arguments<Options>): Promise<void> => {
+  const { privateKey, host, token, query, format } = argv;
+  const options: ConnectionOptions = {};
+  if (privateKey) {
+    // FIXME: This is a hack due to js-tableland's restrictive use of provider
+    // See: https://github.com/tablelandnetwork/js-tableland/issues/22
```
Comment (sanderpick) [inline src/commands/read.ts:37]: still needed?

File: src/commands/receipt.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    type: "string",
+    description: "Transaction hash",
+  }) as yargs.Argv<Options>;
+
+export const handler = async (argv: Arguments<Options>): Promise<void> => {
+  const { hash, host, privateKey } = argv;
+  const options: ConnectionOptions = {};
+  if (privateKey) {
+    // FIXME: This is a hack due to js-tableland's restrictive use of provider
+    // See: https://github.com/tablelandnetwork/js-tableland/issues/22
```
Comment (sanderpick) [inline src/commands/receipt.ts:30]: here too

**Review by sanderpick — APPROVED** [review]
very nice! 💪 left two comments

**Review by joewagner — APPROVED** [review]
All looks good to me!  It builds locally if I `npm link`.  Do we want to update the dep for @tableland/sdk to "^0.1.0"?  Or do that later?

File: src/commands/read.ts (hunk 1 above)
Comment (carsonfarmer) [reply src/commands/read.ts:37]: Yeh unfortunately. I didn't end up tacking this in js-tableland. But maybe now is the time to just fix this.

File: src/commands/receipt.ts (hunk 2 above)
Comment (carsonfarmer) [reply src/commands/receipt.ts:30]: Same same.

File: src/commands/read.ts (hunk 1 above)
Comment (carsonfarmer) [reply src/commands/read.ts]: Nope, not needed because I fixed in @tableland/sdk!

File: src/commands/receipt.ts (hunk 2 above)
Comment (carsonfarmer) [reply src/commands/receipt.ts]: Nope, not needed because I fixed in @tableland/sdk!

**Pull request merged**